nixos/modules/programs/espanso-capdacoverride/default.nix

```
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+{
+  meta = {
+    maintainers = with maintainers; [ pitkling ];
+  };
+
+  options = {
+    programs.espanso.capdacoverride = {
+      enable = (mkEnableOption "espanso-wayland overlay with DAC_OVERRIDE capability") // {
+        default = false;
+        extraDescription = ''
```

```suggestion
        description = ''
```
the default is already false

---

done